# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../btrfs-progs/ppc64.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./print-tree.c.vanilla	2009-10-23 11:45:06.000000000 +0000
+++ ./print-tree.c	2009-10-23 11:45:11.000000000 +0000
@@ -493,7 +493,7 @@
 		case BTRFS_DIR_LOG_INDEX_KEY:
 		case BTRFS_DIR_LOG_ITEM_KEY:
 			dlog = btrfs_item_ptr(l, i, struct btrfs_dir_log_item);
-			printf("\t\tdir log end %Lu\n",
+			printf("\t\tdir log end %lu\n",
 			       btrfs_dir_log_end(l, dlog));
 		       break;
 		case BTRFS_ORPHAN_ITEM_KEY:
--- ./disk-io.c.vanilla	2009-10-23 11:44:15.000000000 +0000
+++ ./disk-io.c	2009-10-23 11:44:37.000000000 +0000
@@ -678,7 +678,7 @@
 		   ~BTRFS_FEATURE_INCOMPAT_SUPP;
 	if (features) {
 		printk("couldn't open because of unsupported "
-		       "option features (%Lx).\n", features);
+		       "option features (%lx).\n", features);
 		BUG_ON(1);
 	}
 
@@ -692,7 +692,7 @@
 		~BTRFS_FEATURE_COMPAT_RO_SUPP;
 	if (writes && features) {
 		printk("couldn't open RDWR because of unsupported "
-		       "option features (%Lx).\n", features);
+		       "option features (%lx).\n", features);
 		BUG_ON(1);
 	}
 
--- ./extent-tree.c.vanilla	2009-10-23 11:44:47.000000000 +0000
+++ ./extent-tree.c	2009-10-23 11:44:55.000000000 +0000
@@ -1448,7 +1448,7 @@
 		goto out;
 	if (ret != 0) {
 		btrfs_print_leaf(root, path->nodes[0]);
-		printk("failed to find block number %Lu\n", bytenr);
+		printk("failed to find block number %lu\n", bytenr);
 		BUG();
 	}
 
